npm/mount-utils/README.md

```diff
@@ -2,7 +2,7 @@
 
 > **Note** this package is not meant to be used outside of cypress component testing.
 
-This librares exports some shared types and utility functions designed to build adapters for components frameworks.
```

Comment:
I figured that we should leave this one since it contains info that isn't captured in the docs, and it's a little different from our other adapter packages